### PR TITLE
Added a simple 'getting started' guide

### PR DESCRIPTION
**Pull request by ColinEberhardt**
Just a simple 'hello world' example. This does a good job of highlighting areas that need improvement!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,104 @@
...
+    "version": "stable"
+  }
+}
+```
+
+The manifest tells the container which URL to load initially.
+
+### Starting a local server
+
+The container loads HTML applications over HTTP, so in order to run this demo application you need to start a local server. If you don't already have a prefered tool, yuo can use the node `http-server` package:
```
Comment (BenLambertNcl) [inline README.md]: yuo => you

File: README.md [hunk 2]
```diff
@@ -2,6 +2,104 @@
...
+
+### Running with Electron
+
+In order to run the same application with Electron, you need to install the CLI tool which is part of this project. From within the `packages/api-electron` folder, run the following:
+
+```
+npm install --global --ignore-scripts .
+```
+
+This install the `ssf-electron` command line tool as a global npm package.
```
Comment (BenLambertNcl) [inline README.md]: install => installs

File: README.md (hunk 1 above)
Comment (BenLambertNcl) [reply README.md]: prefered => preferred

**Review by BenLambertNcl — COMMENTED** [review]
LGTM. Just a few typos :+1:

**Comment by ColinEberhardt** [comment]
> LGTM. Just a few typos

My Atom spell checker seems to have stopped working 🙄

**Review by AndyMJones — COMMENTED** [review]
Yeah LGTM too other than the typos

**Comment by ColinEberhardt** [comment]
Spelling fixed :-)

**Review by BenLambertNcl — APPROVED** [review]

**Pull request merged**